employee-rostering/src/main/java/employeerostering/employeerostering/DayOffRequest.java

```diff
@@ -0,0 +1,36 @@
+package employeerostering.employeerostering;
```

Comment:
@ge0ffrey This was discussed yesterday on IRC, and the conclusion was the files should have the license headers. Most likely the reason why the other examples don't have the headers is because they were generated by the workbench.